File: 2003.py
while True:
  try:
      horario = input()
      hora = int(horario[0])
      minuto = int(horario[2:])
      soma = 0
      minuto+=60
      if minuto >= 60:
          hora+=1
          minuto = minuto-60
      if hora > 8:
          soma += 60*(hora-8)
      if hora >= 8 and minuto > 0:
          soma += minuto
      print(f"Atraso maximo: {soma}")
  except EOFError:
    break

File: 2003.cpp
#include <iostream>
using namespace std;
int main(){
    string horario;
    int hora, minuto, soma;
    while(cin >> horario){
        hora = stoi(horario.substr(0, 1));
        minuto = stoi(horario.substr(2, 4));
        soma = 0;
        minuto+=60;
        if(minuto >= 60){
            hora++;
            minuto -= 60;
        }
        if(hora > 8)
            soma += 60*(hora-8);
        if(hora >= 8 && minuto > 0)
            soma += minuto;
        cout << "Atraso maximo: "<< soma << endl;
    }
    return 0;
}
